### DISCUSSION TITLE
Assistants are always selected

### DISCUSSION THREAD

**Discussion by philipbrembeck** [danny-avila/LibreChat#3694] [Q&A] [answered]
Hey there,
thank you very much for maintaining this awesome project!

I'm reaching out with a small issue I've encountered regarding the user experience with Azure OpenAI and Azure Assistants. When users log in, the system currently defaults to Azure Assistants as shown in the screenshot:

<img width="364" alt="Screenshot 2024-08-19 at 11 50 39" src="https://github.com/user-attachments/assets/68641456-f9cf-41e2-b3aa-f1fdc99aa663">

I was wondering if there's a way to set Azure OpenAI as the default selection instead, with Azure Assistants being an option in the dropdown menu for those who want to use it.

Is this customization possible within the current setup? Any advice or guidance on how to achieve this would be greatly appreciated.

I could not find anything corresponding in the docs, sadly.

**Comment by danny-avila** [comment, marked as answer]
There will be an `order` field in the `librechat.yaml` file but for now, you can do this:

```bash
ENDPOINTS=azureOpenAI,azureAssistants,gptPlugins
```

That will set the order for you
